tests/FixerFactoryTest.php

```diff
@@ -307,11 +334,56 @@ public function testSetWhitespacesConfig(): void
         $factory = new FixerFactory();
         $config = new WhitespacesFixerConfig();
 
-        $fixer = $this->prophesize(\PhpCsFixer\Fixer\WhitespacesAwareFixerInterface::class);
-        $fixer->getName()->willReturn('foo');
-        $fixer->setWhitespacesConfig($config)->shouldBeCalled();
+        $fixer = new class($config) implements WhitespacesAwareFixerInterface {
```

Comment:
I wonder if we can have some DRY here. maybe those anon classes can extend one created via `createFixerDouble` ?